packages/ui/src/primitives/tailwind/Select/index.tsx

```diff
@@ -45,7 +45,7 @@ export interface OptionType {
 export interface SelectProps<T = string | number> {
   options: OptionType[]
   width?: 'sm' | 'md' | 'lg' | 'full'
-  inputSizeVariant?: InputProps['variantSize']
+  inputSize?: InputProps['size']
```

Comment:
`inputSize` is just for the height right?
how about we rename it to `height`?

Reply:
updated